Makefile

```diff
@@ -146,6 +146,12 @@ ifeq ($(USE_LOCAL_IMG),true)
 endif
 	@sed -i'' -e 's@image: .*@image: '"${IMG}"'@' ./overlays/dev/manager_image_patch.yaml
 
+# Rebuild pkg/target/target_template_source.go to pull in pkg/target/regolib/src.rego
+target-template-source:
```

Comment:
Can we add invocation of `target-template-source` somewhere in the makefile that makes it very likely it will be executed on build?

Reply:
Sure, I added it as a dependency of `make docker-build`

Reply:
I think `generate:` is the most appropriate place for this dependency. It doesn't fit the original desired guarantee, but fits with the spirit of what we want.

Reply:
Good call - done